sbe/__init__.py

```diff
@@ -258,7 +273,7 @@ def decode(self, val: int) -> List[str]:
             length = FORMAT_SIZES[self.encodingType.primitiveType] * 8
 
         bits = bitstring.Bits(uint=val, length=length)
-        return [self.choices[i].name for i, v in enumerate(bits) if v]
+        return [self.choices[i].name for i, v in enumerate(reversed(bits)) if v]
```

Comment:
What's the motivation for this? [This example](https://www.fixtrading.org/standards/sbe-online/#multi-value-example) doesn't seem like the bits need to be reversed

Reply:
This is purely a matter of how Python's enumerate() orders the bitstring.Bits. From your example, the old code would decode 0x80 as FinancialStatusEnum.Bankrupt because `i, v` would take values `0,True; 1,False;...;7,False`.